.github/workflows/windows.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
       - name: pip-packages
         run: |
-          pip install numpy==1.22.4 cython pybind11 pythran meson ninja pytest pytest-xdist pytest-timeout pooch rich_click click doit pydevtool hypothesis "scipy-openblas32<=0.3.23.293.2"
+          pip install numpy==1.23.5 cython pybind11 pythran meson ninja pytest pytest-xdist pytest-timeout pooch rich_click click doit pydevtool hypothesis "scipy-openblas32<=0.3.23.293.2"
```

Comment:
I guess there's a `distutils` requirement missing on this from-source `pip` install of NumPy in the CI? Can't remember if i.e., disabling the build isolation and using a newer `setuptools` that vendors it (?) is one way to fix. But also not sure if worth it here.

Reply:
The `==1.23.5` wasn't needed. The `cp310` job is oldest versions; the `cp312` can be with the latest numpy release. All green now.